PlayerCore/actions/VRMCoreItemScheduling.swift

```diff
@@ -0,0 +1,19 @@
+//  Copyright 2018, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+
+extension VRMCore {
+    
+    struct StartItemFetching: Action {
```

Comment:
All our actions are `struct` I don't think that we should sacrifice this rule in current case. Just trying to keep everything in the same style.

Reply:
This will bring nested switch cases 
<img width="321" alt="screen shot 2018-12-19 at 8 01 52 pm" src="https://user-images.githubusercontent.com/18140312/50238849-2379b780-03c9-11e9-8fe3-1528428d02a7.png">

Reply:
About `break` - it's just an example. 
About nested switch - what problem does it solve? It's an extra step which doesn't make any logic. As for me it looks oddly.

Reply:
For me it looks ok - everything is type safe here.
In this particular case - no big advantages of enum vs struct. More semantically. It is ok to go with struct if you want.